.github/workflows/ci.yaml

```diff
@@ -236,7 +236,7 @@ jobs:
   test-image-build:
     runs-on: ubuntu-latest
     env:
-      PLATFORMS: linux/amd64
+      PLATFORMS: linux/amd64,linux/arm64
```

Comment:
should we add `linux/amd64,linux/arm,linux/arm64,linux/s390x`?

Reply:
Absolutely. Updated the file to inculcate these platforms.

Reply:
@tao12345666333 I added `linux/arm,linux/s390x` but for some reason that didn't work. On trying locally I observed the following error. Hence, I have removed these platforms from ci.yaml. Please share if you have any recommendations on it. Thank you. 
```
#7 [4/5] RUN luarocks install lua-resty-template
#7 0.119 qemu: uncaught target signal 4 (Illegal instruction) - core dumped
```